views/random-beers.hbs

```diff
@@ -0,0 +1,7 @@
+<h1>A Random Beer</h1>
+
+{{#each ranBeer}}
+{{>beersPartial this}}
+<p>{{this.food-pairing}}</p>
```

Comment:
I think the field name here is `food_pairing`. This will get you an array. You can use #each to loop on all the elements and put them in a list.
```
            <h5>Food Pairing</h5>
            <ul>
                {{#each this.food_pairing}}
                    <li>{{this}}</li>
                {{/each}}
            </ul>
```